.vscode/settings.json

```diff
@@ -1,8 +1,14 @@
 {
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+  "eslint.run": "onSave",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
     "source.fixAll": true
   },
-  "editor.tabSize": 2
+  "editor.tabSize": 2,
+  "i18n-ally.localesPaths": [
+    "apps/front/public/locales",
+    "libs/front/translations/assets/locales"
+  ],
```

Comment:
I'm not sure of this, is I well understand what you are doing :
- emulate a fetch from server of translation (that are inside `libs/front/translations/assets/locales`)
- save the app translation in the front folder : `apps/front/public/locales`

If this is the case, you may want to remove `"libs/front/translations/assets/locales"` from the config here.

Basically, I feel that this `localesPaths` setting should be the same that the one in `i18n` config file

Reply:
good point, thanks!